tokenStorage.js

```diff
@@ -0,0 +1,56 @@
+/**
+* Default storage a token
+**/
+
+var fs = require('fs');
+
+function TokenStorage(storageFilePath) {
+	this.storagePathFile = storageFilePath || __dirname +'/local/token.txt';
```

Comment:
I don't think that default path should be in a folder. Use of some uniquely-named file (i.e., `.vk-token`) in current working directory (see [process.cwd()](https://nodejs.org/api/process.html#process_process_cwd)) would make more sense.